generator/generator.yml

```diff
@@ -361,3 +361,19 @@ modules:
       - picoMobileMIB
       - picoIPv4MIB
       - picoIPv6MIB
+
+  raritan:
+    walk:
+      - sysUpTime
+      - interfaces
+      - 1.3.6.1.4.1.13742.4.1.3.1.5    # unitCpuTemp
```

Comment:
Typically we like to use the strings, rather than the OID numbers here.

Reply:
Done  :)